TBD/student/templates/student/students.html

```diff
@@ -0,0 +1,8 @@
+<h1>Welcome to Student Section</h1>
+
+{%for x in n_s%}
+{{x.id}}
+{{x.Message}}
+{{x.Created}}
+{{x.Updated}}
+{%endfor%}
```

Comment:
I guess it is a basic template and will require major changes as the project start to expand